wp-content/themes/csisjti/assets/_scss/components/_post-typography.scss

```diff
@@ -101,10 +101,12 @@
   ul {
     margin: rem(16px) 0 rem(32px) rem(56px);
     padding-left: 0;
+    line-height: 1.6;
     @extend %font-copy-body-list-lg;
 
     li {
       position: relative;
+      padding-left: 8px;
 
```

Comment:
Please use our rem() function for this. Rems in the browser provide a more responsive design especially as users zoom in and out in the browser. 

Update code to:
`padding-left: rem(8);`